package.json

```diff
@@ -28,6 +28,7 @@
     "check-translation": "node node_modules/@govuk-one-login/di-ipv-cri-common-express/scripts/checkTranslations.js '../../../../src/locales'"
   },
   "dependencies": {
+    "@govuk-one-login/one-login-language-toggle": "^2.0.0",
```

Comment:
Please install packages using specific versions, rather than the floating caret.

Reply:
Hi @merlinc , thank you for your comments so far but this PR is still in draft. When it is ready to review it will be opened and I will let the you know via the collaboration channel.